src/fsharp/FSharp.DependencyManager.Nuget/FSharp.DependencyManager.ProjectFile.fs

```diff
@@ -40,9 +41,18 @@ module internal ProjectFile =
         |> Array.distinct
 
     let findReferencesFromResolutions (resolutions:Resolution array) =
+
+        let equals (s1:string, s2:string) =
+            let s1 = if String.IsNullOrEmpty(s1) then "" else s1
+            let s2 = if String.IsNullOrEmpty(s2) then "" else s2
+            String.Compare(s1, s2, StringComparison.InvariantCultureIgnoreCase) = 0
+
         resolutions
-        |> Array.filter(fun r -> not(String.IsNullOrEmpty(r.NugetPackageId)) &&
-                                 File.Exists(r.FullPath))
+        |> Array.filter(fun r -> not(String.IsNullOrEmpty(r.NugetPackageId) ||
+                                     String.IsNullOrEmpty(r.FullPath)) &&
+                                     String.Compare(r.IsNotImplementationReference, "true", StringComparison.InvariantCultureIgnoreCase) <> 0 &&
```

Comment:
Can this be replaced `not (equals r.IsNotImplementationReference "true")`?

Reply:
yes